src/cogs/jaegeraccounts.py

```diff
@@ -78,15 +79,14 @@ async def _get_accounts(self):
                         from_datetime_str = date_str + "_" + from_time_str
 
                         try:
-                            last_booked_from = datetime.datetime.strptime(from_datetime_str, "%-m/%-d/%Y_%-I:%M%p")
+                            last_booked_from = datetime.datetime.strptime(from_datetime_str, '%m/%d/%Y_%I:%M%p')
                             last_booked_from = last_booked_from.replace(tzinfo=datetime.timezone(datetime.timedelta(hours=utcoffset)))  # Makes datetime object timezone aware
 
                             last_booked_to = None
                             if to_time_str:
                                 to_datetime_str = date_str + "_" + to_time_str
-                                last_booked_to = datetime.datetime.strptime(to_datetime_str, "%-m/%-d/%Y_%-I:%M%p")
+                                last_booked_to = datetime.datetime.strptime(to_datetime_str, "%m/%d/%Y_%I:%M%p")
```

Comment:
I dont really mind, but is there a particular reason for that?
This would make it 'harder' for humans to edit the excel sheet.

Reply:
This is because it doesn't work on Windows. For some reason it did fine with parsing dates in test sheet

Reply:
WTH, why?
And why does it work for me?
Gaaah Microsoft!

Reply:
idk, it gave errors, i googled it and it resulted in "Windows doesnt support" replies
I will change it back with next commits
This was just to let you know what is going on